### PR TITLE
fix: [ANDROAPP-2813] Show full label in every field

### PR DESCRIPTION
**Pull request by Balcan**
[issue](https://jira.dhis2.org/browse/ANDROAPP-2813)

### CODE REVIEW

File: app/src/main/java/org/dhis2/Bindings/ValueTypeExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        ValueType.DATETIME,
+        ValueType.TIME -> context.getString(R.string.enter_value)
+    }
+}
+
+fun Context.valueTypeHintMap(): Map<ValueType, String> {
+    return ValueType.values().map {
+        it to it.toHint(this)
+    }.toMap()
+}
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/Bindings/ValueTypeExtensions.kt]: This looks like a context extension rather than a valueType extension.

Comment (Balcan) [reply app/src/main/java/org/dhis2/Bindings/ValueTypeExtensions.kt]: But it is heavily related to the ValueType. That's why I decided to leave it there.

File: app/src/main/java/org/dhis2/Bindings/ValueTypeExtensions.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        ValueType.DATE,
+        ValueType.DATETIME,
+        ValueType.TIME -> context.getString(R.string.enter_value)
+    }
+}
+
+fun Context.valueTypeHintMap(): Map<ValueType, String> {
+    return ValueType.values().map {
+        it to it.toHint(this)
+    }.toMap()
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/Bindings/ValueTypeExtensions.kt]: now having a map, could we get a crash when doing .get , if it does not match it?

Comment (Balcan) [reply app/src/main/java/org/dhis2/Bindings/ValueTypeExtensions.kt]: Nop, cause it is using its own values

**Comment by JaimeToca** [comment]
could you add to the ticket, to test all the parts that might have side effects :pray:  thanks !

**Pull request merged**